server/src/main/java/dev/slimevr/vr/processor/skeleton/SkeletonConfig.java

```diff
@@ -21,6 +21,9 @@ public class SkeletonConfig {
 		SkeletonConfigValues.class
 	);
 
+	protected boolean[] changedToggles = new boolean[SkeletonConfigToggles.values().length];
+	protected boolean[] changedValues = new boolean[SkeletonConfigValues.values().length];
```

Comment:
Oh also these should use `values` and not `values()`
```suggestion
	protected boolean[] changedToggles = new boolean[SkeletonConfigToggles.values.length];
	protected boolean[] changedValues = new boolean[SkeletonConfigValues.values.length];
```